src/tools/haxelib/Data.hx

```diff
@@ -38,6 +38,7 @@ typedef UserInfos = {
 typedef VersionInfos = {
 	var date : String;
 	var name : String;
+	var downloads : Int;
```

Comment:
I'm concerned about modifying Data.hx because these are the typedefs that the haxelib client uses to communicate with the server.  If we change them it could break remoting on older clients.  (It might be okay, but having a client and server using different versions of the same type does weird things to serialization, and therefore remoting).

We could work around this by creating a new API endpoint: http://lib.haxe.org/api/

I just want to think it through before merging.